### ISSUE TITLE
Modify default DPI

### ISSUE THREAD

**Issue by jpotier** [krayon/ratslap#3]
In the case I assigned all button to send mouse clicks, how can I set a default DPI?

    ./ratslap --modify F4 --colour Black --G6 Button8 --G7 Button9 --G8 Button10 --G9 Button11

My mouse chooses 1000 by default, but I like it at 2500 for gaming:

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
RatSlap v0.2.0 (BUILT: 2016-11-09 10:35:24+0100)
Copyright (C) 2016 Todd Harbour
Linux configuration tool for Logitech mice (currently only G300S)
https://gitlab.com/krayon/ratslap/
Found Logitech G300s (046d:c246) @ 0x24ed3f0
Detaching kernel driver...
Printing Mode: F4
  Colour:              black
  Report Rate:         1000
  DPI #1:               500
  DPI #2:        (DEF) 1000
  DPI #3:              1500
  DPI #4:              2500
  DPI Shift:           500
  Left Click (But1):   Button1
  Right Click (But2):  Button2
  Middle Click (But3): Button3
  G4:                  Button6
  G5:                  Button7
  G6:                  Button8
  G7:                  Button9
  G8:                  Button10
  G9:                  Button11
Attaching kernel driver...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

I know it's in your todo list, but it may be clearer to have an issue for that.

**Comment by krayon** [comment]
@jpotier Yeah, setting the DPI values hasn't been pushed yet.

As a temporary work around given the above, you could obtain the 2500 by assigning a button to DPIUp and cycling through to it, then change the button back.

**Comment by krayon** [comment]
This feature is tracked as https://bugs.qdnx.org/bug/113 ( "Configure DPI #[1-4], DPI Default and DPI Shift values" )

**Comment by jpotier** [comment]
That's exactly what I did : )

**Comment by krayon** [comment]
Fixed in 8033303e2d37e23e7ad92c18e39ac213d5edaf72
